charts/argo-cd/templates/argocd-server/service.yaml

```
@@ -44,3 +44,6 @@ spec:
 {{ toYaml .Values.server.service.loadBalancerSourceRanges | indent 4 }}
 {{- end }}
 {{- end -}}
+{{- if .Values.server.service.externalTrafficPolicy }}
+  externalTrafficPolicy: {{ toYaml .Values.server.service.externalTrafficPolicy }}
```

toYaml seems overkill for me. The field `externalTrafficPolicy` is a simple string.
```suggestion
{{- with .Values.server.service.externalTrafficPolicy }}
  externalTrafficPolicy: {{ . }}
```